pkg/rancher-desktop/utils/version.ts

```diff
@@ -0,0 +1,39 @@
+import { app } from 'electron';
+
+import { spawnFile } from '@pkg/utils/childProcess';
+
+function getProductionVersion() {
+  try {
+    return app.getVersion();
+  } catch (err) {
+    console.log(`Can't get app version: ${ err }`);
+
+    return '?';
+  }
+}
+
+async function getDevVersion() {
+  try {
+    const { stdout } = await spawnFile('git', ['describe', '--tags'], { stdio: ['ignore', 'pipe', 'inherit'] });
+
+    return stdout.trim();
+  } catch (err) {
+    console.log(`Can't get app version: ${ err }`);
+
+    return '?';
+  }
+}
+
+export async function getVersion() {
```

Comment:
Should we have some sort of warning that this function is main process only? This PR doesn't introduce any mistakes, but this file _does_ get imported in the renderer process, so it seems likely we'll mess up in the future.

Reply:
I had considered this issue and didn't really like the approach. The only function that's shared between main and renderer is `parseDocsVersion`. Perhaps, it makes sense to introduce a separate `parsers` util to keep this separated. 

We'll still have the issue of lacking clear separation between main and renderer, but it'll be segmented between different utils. This isn't the first time we've encountered such an issue, perhaps it'll wise for us to consider how we can better promote clear separation in the future 🤔